darts/tests/models/forecasting/test_transformer_model.py

```diff
@@ -110,3 +110,22 @@ def helper_test_pred_length(self, pytorch_model, series):
         def test_pred_length(self):
             series = tg.linear_timeseries(length=100)
             self.helper_test_pred_length(TransformerModel, series)
+
```

Comment:
Could you also add some "dummy" tests instantiating all of the glu variants? The current PR decreases the code coverage of unit tests (🙀 ), I suspect it comes from the fact that most of these classes are not instantiated in the tests.

Reply:
added test